store/store.go

```diff
@@ -211,20 +211,20 @@ func NewUbuntuStoreSnapRepository(cfg *SnapUbuntuStoreConfig, storeID string) *S
 }
 
 // small helper that sets the correct http headers for the ubuntu store
-func (s *SnapUbuntuStoreRepository) applyUbuntuStoreHeaders(req *http.Request, accept string, auther Authenticator) {
+func (s *SnapUbuntuStoreRepository) setUbuntuStoreHeaders(req *http.Request, channel string, auther Authenticator) {
 	if auther != nil {
 		auther.Authenticate(req)
 	}
 
-	if accept == "" {
-		accept = "application/hal+json"
-	}
-	req.Header.Set("Accept", accept)
-
+	req.Header.Set("Accept", "application/hal+json,application/javascript")
```

Comment:
In the later code I see `application/hal+json`  and `application/json` - but not `application/javascript`. A typo?